tox.ini

```diff
@@ -3,7 +3,7 @@ envlist =
   py35-django{111,22}-backend{Basic,Allauth}
   py36-django{111,22}-backend{Basic,Allauth}
   py37-django{111,22}-backend{Basic,Allauth}
-  py38-django22-backend{Basic,Allauth}
+  py38-django{22,3}-backend{Basic,Allauth}
```

Comment:
Should this test with the latest `django-allauth`?

Reply:
No, because this breaks `tox` when testing against Django V1.11.  This helps make `django-invitations` compatible with Django V3 while keeping functionality with version 1.  This avoids the hard decision of EOL's Django V1.11s compatibility in the interim while the maintainer can consider the roadmap moving forward.

Reply:
I meant for that line with Django 2.2 and 3.

Reply:
Ah. Then sure, but since that involves some `tox` syntax of which I'm not fully literate, I figured to leave that for a future pull request.

Reply:
In the interim, testing against N-1 version is fine.